cdap-ui/app/cdap/components/Replicator/Create/Content/Assessment/ConnectivityAssessment/index.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/core/styles/withStyles';
+
+const styles = (): StyleRules => {
+  return {
+    gridWrapper: {
+      height: '100%',
+
+      '& .grid.grid-container.grid-compact': {
+        maxHeight: '100%',
+
+        '& .grid-row': {
+          gridTemplateColumns: '1fr 2fr 1fr 1fr',
+          gridColumnGap: '50px',
+        },
+      },
+    },
+  };
+};
+
+interface IConnectivityAssessmentProps extends WithStyles<typeof styles> {
+  connectivity: any;
```

Comment:
Why any?

Reply:
will unify the typings in later PR. This was still in flux and going to be shared with detail view as well.